src/UdApi.ts

```diff
@@ -26,10 +27,9 @@ export default class Udapi extends NamingService {
   static readonly ZnsRegistryMap = {
     1: 'zil1jcgu2wlx6xejqk9jw3aaankw6lsjzeunx2j0jz',
   };
-  readonly network: number;
 
   constructor(api?: Api) {
-    super();
+    super(api?.network || 1, BlockchainType.ANYCHAIN);
```

Comment:
I don't really like the ambiguous `BlockchainType.ANYCHAIN` here. I could add some separate logic for `UdApi` to figure out which chain the domain actually belongs to. But on the other hand, it would add unnecessary complications for other services. For other services the BlockchainType is always the same and it can be simply hard-coded. 

And we can't really pull the BlockchainType from anywhere, it can only be determined from the domain TLD.

@DeRain do you think it will be ok to leave it as?